src/utils.ts

```diff
@@ -7,7 +7,8 @@ import * as Sentry from '@sentry/browser'
 import { getLogger } from './logger'
 import { InvalidAppId } from './errors'
 
-const fallbackLogo = '../images/fallback-logo.svg'
+const fallbackLogo =
+  'https://arcana-front.s3.ap-south-1.amazonaws.com/fallback-logo.svg'
```

Comment:
Another approach: Use [a data URI](https://css-tricks.com/data-uris/) to embed the image. This way we remove the dependency on S3.